src/figuresClasses.ts

```diff
@@ -1,19 +1,73 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape,
+  color: Color,
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
 
-}
+  constructor(
+    public color: Color,
+    public a:number,
+    public b:number,
+    public c:number,
+  ) {
+    if (a < 0 || b < 0 || c < 0) {
+      throw new Error("can't be less than 0");
+    }
+
+    if (a >= b + c || b >= a + c || c >= a + b) {
+      throw new Error('One of the sides bigger than two others');
+    }
+  }
 
-export class Circle {
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c));
 
+    return +area.toFixed(2);
```

Comment:
I would recommend to use Number() constructor, it's more obvious
```suggestion
    return Number(area.toFixed(2));
```